source/include/core_json.h

```diff
@@ -118,9 +120,8 @@ JSONStatus_t JSON_Validate( const char * buf,
  *
  * @return #JSONSuccess if the queryKey is found and the value output;
  * #JSONNullParameter if any pointer parameters are NULL;
- * #JSONBadParameter if the queryKey is empty, or any subpart is empty, or max is 0;
- * #JSONIllegalDocument if the buffer contents are NOT valid JSON;
- * #JSONMaxDepthExceeded if object and array nesting exceeds a threshold;
+ * #JSONBadParameter if the queryKey is empty, or any subpart is empty, or max is 0,
+ * or an index is too large to convert;
  * #JSONNotFound if the queryKey is NOT found.
  *
  * <b>Example</b>
```

Comment:
The example needs to be updated to not provide the separator character. 
Also, would it be helpful to provide 2 different examples, one for a JSON object and another for JSON array in the code snippet?

Reply:
fixed
More is usually better, but with our time window, I'll demur.